packages/app/src/cli/api/graphql/webhooks/queries/public-api-versions.graphql

```diff
@@ -0,0 +1,7 @@
+query publicApiVersions {
+  publicApiVersions {
+    displayName
+    handle
```

Comment:
Given `handle` is the only thing we use of this query, should we limit what we fetch to that?